--- config.h.default	2014-08-10 06:15:17.031652941 +0900
+++ config.h	2014-08-10 06:17:08.684985093 +0900
@@ -40,6 +40,7 @@
 static const char *gmrun[]     = { "/usr/bin/gmrun",NULL};
 static const char *terminal[]  = { "urxvtc", NULL };
 static const char *twobwm_path = "/usr/local/bin/2bwm";
+static const char *dmenu_run[]     = { "/usr/bin/dmenu_run",NULL};
 ///---Shortcuts---///
 /* Check /usr/include/X11/keysymdef.h for the list of all keys
  * For AZERTY keyboards XK_1...0 should be replaced by :
@@ -157,6 +158,7 @@
     {  MOD ,              XK_Return,     start,             {.com = terminal}},
     {  MOD ,              XK_w,          start,             {.com = menucmd}},
     {  MOD |SHIFT,        XK_w,          start,             {.com = gmrun}},
+    {  MOD ,              XK_p,          start,             {.com = dmenu_run}},
     // Exit or restart 2bwm
     {  MOD |CONTROL,      XK_q,          twobwm_exit,         {.i=0}},
     {  MOD |CONTROL,      XK_r,          twobwm_restart,      {.i=0}},
